index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Standard Code Developers summit</title>
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css"
+    integrity="sha256-46r060N2LrChLLb5zowXQ72/iKKNiw/lAmygmHExk/o=" crossorigin="anonymous" />
+
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+
+  <header>
+    <nav class="media-links">
+      <ul>
+        <li> <a href=""><i class="fab fa-facebook-f"></i></a></li>
+        <li> <a href=""><i class="fab fa-twitter"></i></a></li>
+        <li> <a href="">English</a></li>
+        <li> <a href="">My Page</a></li>
+        <li> <a href="">Logout</a></li>
+      </ul>
+    </nav>
+    <nav>
+
+  
+      <button type="button" onclick="openMenu()"   class="noMenu"><i class="fa fa-bars fa-2x" id="burger" aria-hidden="true"  onclick="openMenu()"></i></button>
```

Comment:
- [ ] It would be good to keep all your JavaScript code outside of the HTML file, so please work on this issue.

_Note:_ Please do it on the entire HTML file.